AirCasting/CreateSessionViews/CreateSessionDetailsView.swift

```diff
@@ -143,10 +155,21 @@ private extension CreateSessionDetailsView {
 
     func getAndSaveStartingLocation() {
         let fakeLocation = CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0)
-        if isIndoor {
-            sessionContext.startingLocation = fakeLocation
+        if sessionContext.sessionType == .fixed {
+            if isIndoor {
+                sessionContext.startingLocation = fakeLocation
+                locationTracker.googleLocation = [PathPoint(location: CLLocationCoordinate2D(latitude: 200.0, longitude: 200.0), measurement: 20.0)]
```

Comment:
Hmm, I'm not sure about using `measurement: 20.0`, could you leave a comment with some explanation why it's hard coded to be 20?

Reply:
I should say how it is so: when I was doing it, I didn't know what to write there, so that's why 20. Could you please tell me what should be there because I don't know really

Reply:
Maybe we could create something like "fakeMeasurement" (similar to fake location) for this, what do you say? Because as far as I see it influence only the color of the dot, am I right? :D

Reply:
Yes, for the point which I have some knowledge. So what measurement point will we choose for fake one?

Reply:
Hmm, dunno, I guess it would be best if we don't use it for anything, even coloring the dot (color should always be green at this point, right?). Then we can do whatever we want with this measurement, just describe it well in comment for future generations :D

Reply:
+1, it's super unclear why the measurement is here. We should aim with the code quality to not be forced to put such explanation comments at all. I also don't like exposing `googleLocation`. 3rd party stuff should be hidden away (but let's not change it since we have app to finish)

Reply:
Like Paweł wrote, let it stay, just put a warning here for future ;)